src/components/widgets/ChartComp.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
Suffixing the component name with "Comp" is redundant. It adds no useful information. I would rather suggest "StockChart". Additionally I suggest putting this and "StockWidget" inside a folder named "StockWidget", to better organize them. `StockWidget.tsx` should then be in `src/components/StockWidget/index.tsx` as it is the exported component, while the other components belonging to StockWidget should be e.g `src/components/StockWidget/StockChart.tsx`